draft-ietf-quic-transport.md

```diff
@@ -1786,9 +1786,24 @@ than it has sent, unless this is a result of a change in the initial limits (see
 Endpoints can use PING frames (type=0x07) to verify that their peers are still
 alive or to check reachability to the peer. The PING frame contains no
 additional fields. The receiver of a PING frame simply needs to acknowledge the
-packet containing this frame. The PING frame SHOULD be used to keep a connection
-alive when a stream is open. The default is to send a PING frame after 15
-seconds of quiescence. A PING frame has no additional fields.
+packet containing this frame.
+
+A PING frame has no additional fields.
+
+The PING frame can be used to keep a connection alive when an application or
+application protocol wishes to avoid the connection from timing out.  An
```

Comment:
Possible dialect difference:  "avoid ... from" doesn't sound right to me.  I'd have expected to see "keep ... from" or "prevent a connection time-out."